contracts/Comet.sol

```diff
@@ -1267,31 +1270,50 @@ contract Comet is CometMainInterface {
 
     /**
      * @notice Get the total number of tokens in circulation
-     * @dev Note: uses updated interest indices to calculate
-     * @return The supply of tokens
+     * @return The supply of tokens (principal)
      **/
     function totalSupply() override external view returns (uint256) {
         (uint64 baseSupplyIndex_, ) = accruedInterestIndices(getNowInternal() - lastAccrualTime);
         return presentValueSupply(baseSupplyIndex_, totalSupplyBase);
     }
 
     /**
-     * @notice Get the total amount of debt
+     * @notice Get the total present value of base in circulation
+     * @dev Note: uses updated interest indices to calculate
+     * @return The present value of tokens
```

Comment:
nit: `present value of supply`?

Reply:
Adding `supply` to the comment, but wdyt about the one for `totalSupply` then?